Makefile

```diff
@@ -223,7 +221,7 @@ build:
 # 'install' uses the host's Golang to place output into $GOPATH/bin
 .PHONY: install
 install:
-	go install -ldflags $(GRAVITY_LINKFLAGS) -tags "$(GRAVITY_BUILDTAGS)" ./tool/tele ./tool/gravity
+	GO111MODULE=on go install -ldflags $(GRAVITY_LINKFLAGS) -tags "$(GRAVITY_BUILDTAGS)" ./tool/tele ./tool/gravity
```

Comment:
This will probably need the `-mod=vendor` too?

Reply:
Right. Will add it.